src/main/java/lotto/model/LottoTicket.java

```diff
@@ -0,0 +1,30 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoTicket {
+
+    private List<Lotto> lottos;
+
+    public LottoTicket(List<Lotto> lottos) {
```

Comment:
`LottoTicket` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private LottoTicket(List<Lotto> lottos) {
```